pkg/pgmodel/ingestor/ingestor.go

```diff
@@ -97,8 +97,12 @@ func (ingestor *DBIngestor) IngestTraces(ctx context.Context, r pdata.Traces) er
 
 			for k := 0; k < spans.Len(); k++ {
 				span := spans.At(k)
-
-				nameID, err := ingestor.tWriter.InsertSpanName(ctx, span.Name())
+				serviceName := "UNKNOWN"
```

Comment:
how about using https://github.com/open-telemetry/opentelemetry-collector/blob/v0.33.0/translator/trace/protospan_translation.go#L38 instead of "UNKOWN"

Alternatively can we use NULL or 0 for the id?